drizzlepac/haputils/product.py

```diff
@@ -261,7 +264,7 @@ def align_to_gaia(
                     refname = (
                         self.product_basename + "_" + catalog_item + "_ref_cat.ecsv"
                     )
```

Comment:
The parenthesis here don't do anythong. `refname` ends up being a string. If this is the intention (which makes sense) can you remove the parenthesis. It's a bit confusing seeing them.